rir/src/ir/BC.cpp

```diff
@@ -100,6 +100,7 @@ BC_NOARGS(V, _)
     case Opcode::br_:
     case Opcode::brtrue_:
     case Opcode::beginloop_:
+    case Opcode::push_context_:
```

Comment:
Why pop_contenxt_ is not considered in these switches?

Reply:
it in the simple instruction macro that creates the entries...